docs/introduction/integration_test.rst

```diff
@@ -394,7 +440,7 @@ Find below a full example for a test suite/case declaration :
   -> teardown_timeout : Parameter run_timeout is not mandatory for test suite setup.
   """
     @pykiso.define_test_parameters(suite_id=1, aux_list=[aux1, aux2], setup_timeout=5)
-    class SuiteSetup(pykiso.BasicTestSuiteSetup):
+    class SuiteSetup(pykiso.GreyTestSuiteSetup):
```

Comment:
Is it grey or gray test? I find both but can't find a preferred spelling, any thought @sebastianpfischer ?

Reply:
Somehow for me, the autocompletion recommend grey so I would recommend "grey"